### ISSUE TITLE
🐛 Bug Report: Health API for database returns status "OK" when database containers are not active

### ISSUE THREAD

**Issue by mbos2** [appwrite/appwrite#1947]
### 👟 Reproduction steps

Steps I do to reproduce this behaviour: 

- Start appwrite locally
- Run official code example from documentation to get database health status
  - It returns `status: OK`
- Stop `appwrite-mariadb` container 
- Run code again
  - Returns `status OK`
  
  Code snipet from [https://appwrite.io/docs/server/health?sdk=nodejs-default#healthGetDB](https://appwrite.io/docs/server/health?sdk=nodejs-default#healthGetDB)

```
  const sdk = require('node-appwrite');

// Init SDK
let client = new sdk.Client();

let health = new sdk.Health(client);

client
    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
    .setProject('PROJECT ID') // Your project ID
    .setKey('PROJECT SECRET) // Your secret API key
;

let promise = health.getDB();

promise.then(function (response) {
    console.log(response);
}, function (error) {
    console.log(error);
});
```  

  ---
  
  NOTE:
  I tried the same code with local express application and Postman. Result was the same.

### 👍 Expected behavior

When database containers are not working, I think that code mentioned in `reproduction steps` should not return status `OK` but either some other status or an error.

### 👎 Actual Behavior

When database container is not working, code mentioned in `reproduction steps` return `status: OK` to me on the console.

### 🎲 Appwrite version

Version 0.10.x

### 💻 Operating system

Windows

### 🧱 Your Environment

OS: Windows 10 Pro

I use docker desktop on windows to run my local [appwrite installation](https://appwrite.io/docs/installation),

I use Svelte for my frontend.

I also tested with Express and Postman locally.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by mbos2** [comment]
I just checked on [https://github.com/appwrite/appwrite/blob/master/app/controllers/api/health.php#L51](https://github.com/appwrite/appwrite/blob/master/app/controllers/api/health.php#L51) before raising the issue 😅 
Does this mean `status: OK` will always be sent in any case?

**Comment by PineappleIOnic** [comment]
Closing this as the relevant fixed have been merged into the codebase 🥳

**Comment by eldadfux** [comment]
@PineappleIOnic  Let's keep it open until the actual release. Adding the label `waiting for release`.

**Comment by eldadfux** [comment]
This fix is now available as part of Appwrite 0.12 which is now officially released. You can learn more at: https://dev.to/appwrite/its-here-announcing-the-release-of-appwrite-012-5c8b
